specs/WebRtcPortConfiguration.md

```diff
@@ -0,0 +1,277 @@
+
+var environment = await CoreWebView2Environment.CreateAsync(
+    browserExecutableFolder: subFolder,
+    userDataFolder: m_userDataFolder,
+    options: options);
+
+OnCreateEnvironmentCompleted(environment);
+```
+
+### C++ Sample
+```cpp
+using namespace Microsoft::WRL;
+
+ScenarioWebRtcUdpPortConfiguration::ScenarioWebRtcUdpPortConfiguration(AppWindow* appWindow)
+    : m_appWindow(appWindow), m_webView(appWindow->GetWebView())
+{
+    // Navigate to a demo page that will trigger WebRTC usage.
+    m_demoUri = L"https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/";
+    CHECK_FAILURE(m_webView->Navigate(m_demoUri.c_str()));
+
+    // If we navigate away from the demo page, turn off this scenario.
+    CHECK_FAILURE(m_webView->add_ContentLoading(
+        Callback<ICoreWebView2ContentLoadingEventHandler>(
+            [this](ICoreWebView2* sender, ICoreWebView2ContentLoadingEventArgs* /*args*/)
+                -> HRESULT
+            {
+                wil::unique_cotaskmem_string uri;
+                sender->get_Source(&uri);
+                if (uri.get() != m_demoUri)
+                {
+                    m_appWindow->DeleteComponent(this);
+                }
+                return S_OK;
+            })
+            .Get(),
+        &m_contentLoadingToken));
+}
+
+ScenarioWebRtcUdpPortConfiguration::~ScenarioWebRtcUdpPortConfiguration()
+{
+    CHECK_FAILURE(m_webView->remove_ContentLoading(m_contentLoadingToken));
+}
+
+```
+### C# Sample
+```csharp
+using Microsoft.Web.WebView2.Core;
+using System;
+
+class ScenarioWebRtcUdpPortConfiguration
+{
+    private readonly AppWindow _appWindow;
+    private readonly CoreWebView2 _webView;
+    private readonly string _demoUri;
+    private EventRegistrationToken _contentLoadingToken;
+
+    public ScenarioWebRtcUdpPortConfiguration(AppWindow appWindow)
+    {
+        _appWindow = appWindow;
+        _webView = appWindow.GetWebView();
+
+        // Navigate to a demo page that will trigger WebRTC usage.
+        _demoUri = "https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/";
+        _webView.Navigate(_demoUri);
+
+        // If we navigate away from the demo page, turn off this scenario.
+        _webView.ContentLoading += WebView_ContentLoading;
+    }
+
+    private void WebView_ContentLoading(object? sender, CoreWebView2ContentLoadingEventArgs e)
+    {
+        var uri = _webView.Source;
+        if (uri != _demoUri)
+        {
+            _appWindow.DeleteComponent(this);
+        }
+    }
+
+    ~ScenarioWebRtcUdpPortConfiguration()
+    {
+        _webView.ContentLoading -= WebView_ContentLoading;
+    }
+}
+```
+
+# API Details
+## C++  
+```
+/// Additional options used to create WebView2 Environment to manage WebRTC UDP port range configuration.
+[uuid(0eebe393-8dcf-5bc5-a15d-d862088242e9), object, pointer_default(unique)]
+interface ICoreWebView2StagingEnvironmentOptions10 : IUnknown {
```

Comment:
For this IDL in the document we need to show the expected stable API. In this case just remove 'Staging' / 'Experimental' from interface names in this IDL should be fine.